frontend/test/metabase/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -168,6 +168,18 @@ describe("scenarios > embedding > dashboard parameters", () => {
       cy.get(".ScalarValue")
         .invoke("text")
         .should("eq", "2,500");
+
+      // embedding with locale metabase#22561
+      cy.url().then(url => {
+        cy.visit({
+          url,
+          qs: {
+            locale: "de-DE",
+          },
+        });
+      });
+
+      cy.findByText("Angetrieben von");
```

Comment:
I piggy bag this test here as this is the first file in the `embedding` test directory, and I remember @nemanjaglumac told me to not create a new test if not necessary as there is an overhead.

![image](https://user-images.githubusercontent.com/1937582/167813006-45e66d7c-aa0b-46ac-9efc-d807c8952eb8.png)

By the way, I'm not sure if I should add the locale test to each of the embedding test files? What do you think @nemanjaglumac ?